src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## **Unreleased**
 * FEATURE: Multitool SARIF rewrite accepts `remove` parameter[#2160](https://github.com/microsoft/sarif-sdk/pull/2160)
+* BREAKING: Remove command `export-validation-docs` and add `export-validation-rules` command can export markdown file [#2156](https://github.com/microsoft/sarif-sdk/pull/2156)
```

Comment:
>Remove command `export-validation-docs` and add `export-validation-rules` command can export markdown file [#2156](https://github.com/microsoft/sarif-sdk/pull/2156) [](start = 12, length = 164)

Remove command `export-validation-docs` and extend `export-validation-rules` command to export markdown file [#2156](https://github.com/microsoft/sarif-sdk/pull/2156)